inject-mathjax.js

```diff
@@ -0,0 +1,149 @@
+// const system = require ('system');
+const fs = require ('fs');
+const puppeteer = require ('puppeteer');
+
+
+// function getPath() {
+//   const path = fs.absolute(system.args[0])
+//   const inx = path.lastIndexOf('/')
+//   return fs.absolute(path.slice(0, inx) + '/..')
+// }
+
+// if (system.args.length !== 3 && system.args.length !== 4) {
+//   console.log('Usage: typeset.js FILE STYLE [OUTPUT]')
+//   browser.close();
+//   console.log("Status: " + status);
+// }
+// const url = system.args[1];
+const url = 'file:///mnt/c/Users/Thomas/Openstax/bakedpdf/u-physics-one-unit.xhtml';
```

Comment:
Can this be pulled in from the commandline args? Maybe https://www.npmjs.com/package/commander or https://www.npmjs.com/package/yargs